wrappers/ortools.rb

```diff
@@ -514,8 +514,7 @@ def parse_output(vrp, _services, points, _matrix_indices, _cost, _iterations, ou
 
       collected_indices = []
       vehicle_rest_ids = Hash.new([])
-      solution_cost = Models::CostDetails.new({})
-      {
+      OptimizerWrapper.parse_result(vrp, {
```

Comment:
I would prefer to put the hash within a variable and then transmit it to the method : `OptimizerWrapper.parse_result(...)`

Having a multi-line argument is pretty hard to understand